modules/aws/acme_certificate/variables.tf

```diff
@@ -0,0 +1,30 @@
+variable "email_address" {
+  description = "Email address to register the Certificate"
+  type        = string
+}
+
+variable "aws_hosted_zone_id" {
+  description = "Hosted zone Id of the domain"
+  type        = string
+}
+
+variable "base_domain_name" {
+  description = "Domain without www. or https to create a certificate for"
```

Comment:
Is this description correct?

Reply:
Rejigged the description and added validation to make it easier to read and use.